index.bs

```diff
@@ -2669,10 +2674,18 @@ The <dfn export for=commands>browser.createUserContext</dfn> command creates a
 
 <div algorithm="remote end steps for browser.createUserContext">
 
-The [=remote end steps=] are:
+The [=remote end steps=] with |command parameters| are:
 
 1. Let |user context| be a new [=user context=].
 
+1. If |command parameters| [=map/contains=] "<code>proxy</code>":
+
+   1. Let |proxy| be [=deserialize as a proxy=] with |command parameters|["<code>proxy</code>"].
```

Comment:
I think it should be `[=trying=] to [=deserialize as a proxy=]`

Reply:
generally, it seems to be used for validation only in the classic spec and validation is covered by CDDL?

Reply:
fair point. Done.